etna/datasets/datasets_generation.py

```diff
@@ -186,3 +186,88 @@ def generate_from_patterns_df(
     df["timestamp"] = pd.date_range(start=start_time, freq=freq, periods=periods)
     df = df.melt(id_vars=["timestamp"], value_name="target", var_name="segment")
     return df
+
+
+def generate_hierarchical_df(
+    periods: int,
+    n_segments: List[int],
+    freq: str = "D",
+    start_time: str = "2000-01-01",
+    ar_coef: Optional[list] = None,
+    sigma: float = 1,
+    random_seed: int = 1,
+) -> pd.DataFrame:
+    """Create DataFrame with hierarchical structure and AR process data.
```

Comment:
From method name and docstring I can't understand how many levels are generated during this method.

Reply:
Add `n_levels` parameter to make it more explicit